cdk_emqx_cluster/cdk_emqx_cluster_stack.py

```diff
@@ -329,7 +401,7 @@ def setup_emqx(self, N):
             userdata_init.add_commands(emqx_user_data)
             multipartUserData = ec2.MultipartUserData()
             multipartUserData.add_part(
-                ec2.MultipartBody.from_user_data(userdata_hostname))
+                ec2.MultipartBody.from_user_data(persistent_config))
```

Comment:
Just curious: on line 219 persistentConfig is in camel case, and here it's in snake case. Is it a typo or intentional?

Reply:
No, it was not intentional... I ended up using two different conventions in the different functions, since the casing appears to be mixed through the file. :see_no_evil:
I'll change this one to camel case.